src/plants/PlantSelectField.js

```diff
@@ -105,75 +116,58 @@ class PlantSelectField extends React.Component {
     const { searchTerm, showResults, isLoading } = this.state;
 
     return (
-      <ClickAwayListener onClickAway={this.closeResults}>
-        <div className={classes.selectWraper}>
-          {selectedPlant && (
-            <List component={Paper}>
-              <PlantItem
-                plant={selectedPlant}
-                classes={classes}
-                deselectPlant={this.deselectPlant}
-              />
-            </List>
+      <>
+        <Autocomplete
+          value={selectedPlant}
+          onChange={(e, newSelectedPlant) => this.selectPlant(newSelectedPlant)}
```

Comment:
Essa parte esta selecionando a planta adequadamente devido `newSelectedPlant` ser um item de `edges` [(coisa do relay)](https://stackoverflow.com/questions/42622912/in-graphql-whats-the-meaning-of-edges-and-node#42623708)

```suggestion
          onChange={(e, {node: newSelectedPlant}) => this.selectPlant(newSelectedPlant)}
```

essa alteração vai quebrar outras partes ai do seu codigo onde vc ta usando `value.node` e `option.node` etc... se tiver duvidas me da um toque

Reply:
li o link que vc mandou e foi muito bom, obrigado! :100: 
pelo o que eu entendi, o objetivo é parar de ficar mandando objetos com a key node pra lá e pra cá, certo?

pra isso eu modifiquei as options pra receber o objeto desestruturado e todo o resto do componente não precisa lidar com isso.

antes:
`options={plants}`

depois:
`options={plants.map(({ node: plant }) => plant)}`

tá no caminho certo ou entendi errado? hahah